src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public class Task {
+    protected String description;
+    protected boolean isCompleted;
```

Comment:
Might want to change the name of this boolean to isTaskCompleted for better understandability!